pyproject.toml

```diff
@@ -64,6 +64,9 @@ python-dateutil = "^2.9.0"
 sortedcollections = "^2.1.0"
 typing-extensions = "^4.1.0"
 tldextract = "^5.1.2"
+cookiecutter = "^2.6.0"
+typer = {extras = ["all"], version = "^0.12.3"}
+inquirer = "^3.3.0"
```

Comment:
This was added because JS crawlee uses the original version, but it doesn't have type stubs, so mypy complains. I think we can remove it if @B4nan agrees and just use whatever `typer` offers

Reply:
Look like neither `typer` nor `rich` support selection from list, which is a shame. So we're stuck with `inquirer` for now.